website/docs/events/brainstorming-2023-06-22.md

```diff
@@ -0,0 +1,63 @@
+# Brainstorming: Education in the Digital Age
+
+Good education is always a challenge, even more so in the digital age, where information is readily available and the potential of automation is disrupting existing methods.
+Questions come about:
+
+* What are good digital education materials?
+* Which materials should be used live (during a class, lecture) or which should be used offline?
```

Comment:
```suggestion
* Which materials should be used live (during a class, lecture) and which should be used offline?
```
This qurstion is a bit strange. It's as if you have a bunch of materials and you delegate some of them for live usage and the rest are left for offline. I'd reword it to "How to design materials for live usage and for individual study. Should they be separate?"